pelican-bootstrap3/templates/includes/sidebar-images.html

```diff
@@ -7,6 +7,9 @@
         {% for image in SIDEBAR_IMAGES %}
             <img width="100%" class="img-thumbnail" src="{{ image }}"/>
         {% endfor %}
+        {% for name, attributes in SIDEBAR_IMAGES_WITH_ATTRIBUTES.iteritems() %}
+            {% if attributes.href is defined %}<a href="{{ attributes.href }}">{% endif %}<img width="{% if attributes.width is defined %}{{ attributes.width }}{% else %}100%{% endif %}" class="{% if attributes.class is defined %}{{ attributes.class }}{% else %}img-thumbnail{% endif %}" src="{{ attributes.src }}"/>{% if attributes.href is defined %}</a>{% endif %}
```

Comment:
Inlined conditionals make things very difficult to read and maintain.  Separate out the conditionals, and assign results to variables.  Then inline the variables.